x-pack/plugin/transform/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/transform/integration/TransformUsageIT.java

```diff
@@ -72,6 +72,8 @@ public void testUsage() throws Exception {
 
         Request getRequest = new Request("GET", getTransformEndpoint() + "test_usage/_stats");
         Map<String, Object> stats = entityAsMap(client().performRequest(getRequest));
+        // temporary debug logs for https://github.com/elastic/elasticsearch/issues/52931
+        logger.info("test_usage/_stats response: [{}]", stats);
```

Comment:
Just double-checking: is `stats` object of type `Map` actually going to be printed as map by the logger?

Reply:
yes, it is. I checked that as I had the same doubt. ;-)